training/cifar/cifar10_deepspeed.py

```diff
@@ -159,7 +160,7 @@ def imshow(img):
 
 # get some random training images
 dataiter = iter(trainloader)
-images, labels = dataiter.next()
+images, labels = next(dataiter)

```

Comment:
Can you please explain this change? I don't understand the difference.

Reply:
From pytorch 1.13 the old usage is deprecated (https://stackoverflow.com/questions/74289077/attributeerror-multiprocessingdataloaderiter-object-has-no-attribute-next).   So to make this work on latest Pytorch we need to change this line.